lib/harmonization.py

```diff
@@ -425,7 +436,7 @@ def main(self):
         # Scale_L{i}.nii.gz of <= {N_shm during template creation} are present only
         elif self.N_shm==-1 and self.process:
             for i in range(0,8,2):
-                if os.path.isfile(os.path.join(self.templatePath, f'Scale_L{i}_b{self.bshell_b}.nii.gz')):
+                if os.path.isfile(os.path.join(self.templatePath, f'Scale_L{i}.nii.gz')):
                     self.N_shm= i
```

Comment:
Since this block was copied over from pnlbwh/multi-shell-dMRIharmonization, the `bshell_b` reference was forgotten to remove.